modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/OrganizationResourceDTOConverter.java

```diff
@@ -109,6 +109,9 @@ public Organization toDTO(
 			return null;
 		}
 
+		Organization parentOrganizationDTO = toDTO(
```

Comment:
this looks strange... why is needed?

Reply:
@nhpatt We get a compile error because now Organization has a toDTO method

Reply:
Oh, I see, I though variable name was the same as property